### ISSUE TITLE
Trouble Connecting with Lego Boost from Windows 10 

### ISSUE THREAD

**Issue by johoul** [LLK/scratch-link#196]
Just trying to setup my kid's new Lego Boost kit for use with scratch.

When I go to the Lego Boost section of  scratch.mit.edu and Click "start searching" I expect to see a representation of the Boost Block connecting (according to various youtube videos) 

Instead it reverts to the "1. Check you have  Scratch Link Running. 2. Check that Bluetooth in enabled." dialog. 

Cycle repeats each time I click "start searching".

This happens for both my Win 10 laptops ,  one running WIndows 10 home 21H1 , one running Windows 10 Enterprise 1909

Tried both Chrome and Firefox and with Scratch Link downloaded from Windows Store and its own website.

Both laptops can see Move Hub If I try to connect with Bluetooth directly but I haven't paired to avoid possible conflict. (I tried connecting while paired in Windows previously but it didn't work either)

Any help gratefully accepted

**Comment by BryceLTaylor** [comment]
Please use the forums (scratch.mit.edu/discuss) for general troubleshooting from users in the Scratch community.  Github issues are for specific issues in the code.

**Comment by johoul** [comment]
Thanks Bryce - I have tried as you suggested but no useful responses yet.  Worryingly, I've seen many people complain in those forums that Scratch-link isn't working for their Boost either. I've also seen similar complaints on YouTube demos of Boost working with Scratch that it not working that well anymore. I've asked the author of one particular high quality video if his system still works ok but I've yet to hear back. 

I realise how much effort goes into trying to keep up with the various closed-shop producers and I really am appreciative of how great Scratch is.  I'm just worried something has gone wrong - maybe the recent firmware update being pushed onto the Boost hubs might be causing Scratch-link to malfunction. I naively downloaded some developing tools to see if I could get a look under the hood of Scratch-link but unsurprisingly didn't get very far (i.e couldn't even get the bonnet open!). Anyways, I did notice in  documentation\TestPlans that Lego Boost isn't mentioned in the various test scenarios.   Is this right? Given that Lego Boost has a separate section in the extensions area of Scratch 3.0  maybe it should also be included in various testing scenarios for Scratch-link (if that is indeed what TestPLans refers to)?

Anyways, whatever the problem is, hopefully it will be resolved soon -
